index.tsx

```diff
@@ -2,6 +2,7 @@ import { IAugmentedJQuery, IComponentOptions } from 'angular'
 import fromPairs = require('lodash.frompairs')
 import mapValues = require('lodash.mapvalues')
 import NgComponent from 'ngcomponent'
+import * as PropTypes from 'prop-types'
```

Comment:
Is there a way we can avoid this dependency?

Reply:
we're only truly dependent on the types here, we don't use any of the actual code. In order to use the Requireable type on line 65 (which I needed to make the code compile) we need to import this typing. If we can reference the type without importing the library I'm 100% down. Perhaps a reference path to the @types relative dir?